Tests/Tests/BundleSpec.swift

```diff
@@ -26,12 +26,6 @@ class BundleSpec: QuickSpec {
                 expect(bundleInfo.appVersion).to(equal("1.2.3"))
             }
 
-            it("should return a valid inAppSdkVersion") {
-                // swiftlint:disable:next line_length
-                let semverRegex = #"^(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$"#
-                expect(bundleInfo.inAppSdkVersion).to(match(semverRegex))
```

Comment:
Hmm, let's keep this test maybe. In case someone (or some script in the future) puts an incorrect version format or makes a typo. I'm not sure where to put it though. Maybe in a new file `ConstantsSpec` or `EnvironmentSpec`?

Reply:
Added here [a7168f6](https://github.com/rakutentech/ios-inappmessaging/pull/288/commits/a7168f69f76364aa7ea89b156292d71eeb8bf880).